libraries/chain/include/eosio/chain/exceptions.hpp

```diff
@@ -196,6 +196,8 @@ namespace eosio { namespace chain {
                                     3040013, "Transaction is too big" )
       FC_DECLARE_DERIVED_EXCEPTION( unknown_transaction_compression, transaction_exception,
                                     3040014, "Unknown transaction compression" )
+      FC_DECLARE_DERIVED_EXCEPTION( bandwith_confirmed, transaction_exception,
```

Comment:
This name is confusing. The question appears: "If bandwidth was confirmed, why throw an exception?"
Please, rename to `bandwidth_already_confirmed`.